plugins/techdocs/src/reader/components/TechDocsReaderPageContent/dom.tsx

```diff
@@ -81,9 +81,26 @@ export const useTechDocsReaderDom = (
       if (isMobileMedia) {
         element.style.top = '0px';
       } else {
-        const domTop = dom.getBoundingClientRect().top ?? 0;
+        let domTop = dom.getBoundingClientRect().top ?? 0;
```

Comment:
How about adding a class to the `TechDocsReaderPage` component and comparing the dom top with it?
Testing this example locally seems to work as expected on both the standard and catalog pages:

_plugins/techdocs/src/reader/components/TechDocsReaderPage/TechDocsReaderPage.tsx_

```diff
const TechDocsReaderPage = (props: TechDocsReaderPageP
rops) => {
   return (
     <TechDocsReaderPageProvider entityRef={entityRef}>
       {({ metadata, entityMetadata, onReady }) => (
-        <Page themeId="documentation">
-          {children instanceof Function
-            ? children({
-                entityRef,
-                techdocsMetadataValue: metadata.value,
-                entityMetadataValue: entityMetadata.value,
-                onReady,
-              })
-            : children}
-        </Page>
+        <div className="techdocs-reader-page">
+          <Page themeId="documentation">
+            {children instanceof Function
+              ? children({
+                  entityRef,
+                  techdocsMetadataValue: metadata.value,
+                  entityMetadataValue: entityMetadata.value,
+                  onReady,
+                })
+              : children}
+          </Page>
+        </div>
```
_plugins/techdocs/src/reader/components/TechDocsReaderPageContent/dom.tsx_
```diff
       if (isMobileMedia) {
         element.style.top = '0px';
       } else {
-        const domTop = dom.getBoundingClientRect().top ?? 0;
+        const page = document?.querySelector('.techdocs-reader-page');
+        const pageTop = page?.getBoundingClientRect().top ?? 0;
+        let domTop = dom.getBoundingClientRect().top ?? 0;
+
         const tabs = dom.querySelector('.md-container > .md-tabs');
         const tabsHeight = tabs?.getBoundingClientRect().height ?? 0;
+
+        if (domTop < pageTop) {
+          domTop = pageTop;
+        }
+
         element.style.top = `${Math.max(domTop, 0) + tabsHeight}px`;
       }
```